package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@dapplion/benchmark",
-  "version": "0.2.6-rc.2",
-  "repository": "git@github.com:dapplion/benchmark.git",
-  "author": "dapplion <35266934+dapplion@users.noreply.github.com>",
+  "name": "@chainsafe/benchmark",
+  "version": "0.1.0.rc-0",
```

Comment:
```suggestion
  "version": "1.0.0",
```

Reply:
As it's a new package I would prefer to start with 0 versioning scheme. 

https://0ver.org/

Reply:
its not a new package though and it was forked from 1.0.0.... if anything it should be `1.1.0` if this is a minor and `2.0.0` if its a major which i dont think it is....

`1.1.0` feels like the right number here

Reply:
and no RC